### DISCUSSION TITLE
Cylindrical volumes?

### DISCUSSION THREAD

**Discussion by paulmelis** [RenderKit/ospray#562] [Q&A]
I noticed #261 talked about new types of volumes, including cylindrical datasets. Has there been any progress on this by any chance?

**Comment by paulmelis** [comment]
Or would it make more sense to enter a request for cylindrical volume support with OpenVKL?

**Reply by johguenther** [reply]
We internally track this as feature request for VKL.

**Reply by paulmelis** [reply]
Ah, great!
